notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<p>Tutaj po połączeniu wszystkich podmiotów samorządowych powinna wyjść bardziej wyraźna różnica</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>